app/controllers/short_url_requests_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
   end
 
   def show
+    @existing_redirect = Redirect.where(from_path: @short_url_request.from_path).first
```

Comment:
Can the be multiple existing requests?  The ticket says "If there are other requests for this short URL, display a list of each request and the following associated fields:".